spec/services/mail_login_token_service_spec.rb

```diff
@@ -14,9 +14,10 @@
 
     describe '#execute' do
       it 'generates new login token for user' do
+        user.regenerate_login_token
```

Comment:
Why this call to `regenerate_login_token`? The check below should be checking whether a `original_login_token` is set by the service. i.e., the change should be from `nil` to some `non-nil` value, which is covered by the `to(change ...)` call.